src/translations/en.json

```diff
@@ -75,6 +75,11 @@
   "BookingDateRangeFilter.labelPopup": "Dates",
   "BookingDateRangeFilter.labelSelectedPlain": "{dates}",
   "BookingDateRangeFilter.labelSelectedPopup": "{dates}",
+  "BookingDateRangeLengthFilter.labelPlain": "Dates",
+  "BookingDateRangeLengthFilter.labelPopup": "Dates",
+  "BookingDateRangeLengthFilter.labelSelectedPlain": "{dates}",
+  "BookingDateRangeLengthFilter.labelSelectedPopup": "{dates}",
+  "BookingDateRangeLengthFilter.minDurationLabel": "Minimum length",
```

Comment:
This is a new translation key. Should this line be added to the other translations as well?

Reply:
@Gnito See the questions above ^

Reply:
I'm not sure what you mean. Options are currently in marketplace-custom-config.js to make it easier to change them on the fly (especially, with category and amenities it is useful). However, since this is not about those, it's probably best if this title-label is here in the translation file.

Reply:
Or did you mean that es, de, fr, should be changed too?

That's happening through WTI at some point (when someone has time to take care of FTW and ask new translations from translators).

Reply:
Yeah, that's what I meant. This is a new translation key, but there's only the English text.

Reply:
FTW defaults to en.json if the key is completely missing from other translation files - so, if the correct translations are not quick to collect, you can leave those as they are.

Reply:
I.e. don't add English translations to fr/de/es.json files.